apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -259,12 +283,17 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
         }
     };
 
-    /**
-     * Set the default multiple invites option.
-     */
-    useEffect(() => {
-        setConfigureMode(MultipleInviteMode.MANUAL);
-    }, [ ]);
+    // Validate the input string is an email address.
+    const validateEmail = (emailList: string[]) => {
+
+        const emailValidation: boolean = FormValidation.email(emailList[emailList.length-1]);
+
+        if (!emailValidation) {
+            setIsEmailDataError(true);
+            setEmailDataError("Enter a valid email address");
```

Comment:
Shall we localize this string?

Reply:
Fixed with 9fd857accd3f35459c4b56b50a2d17744b271f36.